packages/docs/content/4_utilities/2_layout-grid.mdx

```diff
@@ -0,0 +1,249 @@
+
+<EmbeddedExample>
+    <section class="ds-l-container example-grid">
+        <div class="ds-l-row">
+            <div class="ds-l-col">Equal</div>
+            <div class="ds-l-col">Equal</div>
+            <div class="ds-l-col">Equal</div>
+            <div class="ds-l-col">Equal</div>
+        </div>
+    </section>
+</EmbeddedExample>
+
+## Column spanning
+
+Layout regions can span any number of columns and are separated by the grid gutter. Layouts can change at any of the designated breakpoints.
+
+Specify the number of columns (1-12) to span using `.ds-l-col--*` classes. These classes will span `*` columns. So, if you want two elements that each span half the row, you would use `.ds-l-col--6`
+
+<EmbeddedExample>
+    <section class="ds-l-container example-grid">
+        <div class="ds-l-row">
+            <div class="ds-l-col--6">6 columns</div>
+            <div class="ds-l-col--6">6 columns</div>
+        </div>
+    </section>
+</EmbeddedExample>
+
+## Natural width columns
+
+Use a `.ds-l-col--auto` class to size a column to the natural width of its content.
+
+<EmbeddedExample>
+    <section class="ds-l-container example-grid">
+        <div class="ds-l-row">
+            <div class="ds-l-col--auto">1 of 3</div>
+            <div class="ds-l-col--auto">A little bit longer text</div>
+            <div class="ds-l-col--auto">3 of 3</div>
+        </div>
+    </section>
+</EmbeddedExample>
+
+## Nested grids
+
+<EmbeddedExample>
+    <section class="ds-l-container example-grid">
+        <div class="ds-l-row">
+            <div class="ds-l-col--9">
+            Level 1
+                <div class="ds-l-row">
+                    <article class="ds-l-col--4">Level 2</article>
+                    <article class="ds-l-col--4">Level 2</article>
+                    <article class="ds-l-col--4">Level 2</article>
+                </div>
+            </div>
+            <div class="ds-l-col--3">Level 1</div>
+        </div>
+    </section>
+</EmbeddedExample>
+
+## Offsetting columns
+
+The page content does not always need to span across 12 columns; it can occupy a smaller region in the center of the page. This example shows an 8-column layout region with an offset of 2 columns on both sides.
+
+Offset columns using the [margin auto utility class](/utilities/margin):
+
+<EmbeddedExample>
+    <section class="ds-l-container example-grid">
+        <div class="ds-l-row">
+            <div class="ds-l-col--8 ds-u-margin-left--auto ds-u-margin-right--auto">8 columns, offset</div>
+        </div>
+    </section>
+</EmbeddedExample>
+
+## Alignment
+
+Align columns horizontally or vertically using [flexbox utility classes](/utilities/flexbox). To align columns horizontally, use the `justify-content` utility class. To align columns vertically, use the `align-items` utility class.
+
+<EmbeddedExample>
+    <section class="ds-l-container example-grid">
```

Comment:
I just noticed the indentation is different from our other `prettier`-formatted files. The matching pattern seems right for our `yarn prettier` command, so I don't know why it doesn't seem to be formatting these files.

Reply:
Not sure what happened, but I updated the prettierignore then ran `yarn prettier` again and the files seem to be formatted now